webapp/channels/src/components/help/formatting.tsx

```diff
@@ -0,0 +1,719 @@
+                                    <code>{'```'}</code>
+                                </td>
+                                <td>
+                                    <pre className='Help__code-preview'>{'Code block'}</pre>
+                                </td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </section>
+
+                <section className='Help__section'>
+                    <h2>
+                        <FormattedMessage
+                            id='help.formatting.syntax.title'
+                            defaultMessage='Syntax Highlighting'
+                        />
+                    </h2>
+                    <p>
+                        <FormattedMessage
+                            id='help.formatting.syntax.description'
+                            defaultMessage='To add syntax highlighting, type the language to be highlighted after the ``` at the beginning of the code block. Mattermost also offers four different code themes (GitHub, Solarized Dark, Solarized Light, Monokai) that can be changed in <b>Settings > Display > Theme > Custom Theme > Center Channel Styles > Code Theme</b>.'
+                            values={{
+                                b: (chunks: React.ReactNode) => <b>{chunks}</b>,
+                            }}
+                        />
+                    </p>
+                    <p>
+                        <FormattedMessage
+                            id='help.formatting.syntax.example_label'
+                            defaultMessage='Example:'
+                        />
+                    </p>
+                    <table className='Help__table'>
+                        <thead>
+                            <tr>
+                                <th>
+                                    <FormattedMessage
+                                        id='help.formatting.table.text_entered'
+                                        defaultMessage='Text Entered'
+                                    />
+                                </th>
+                                <th>
+                                    <FormattedMessage
+                                        id='help.formatting.table.how_it_appears'
+                                        defaultMessage='How it appears'
+                                    />
+                                </th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr>
+                                <td>
+                                    <code>{'```go'}</code><br/>
+                                    <code>{'package main'}</code><br/>
+                                    <code>{'import "fmt"'}</code><br/>
+                                    <code>{'func main() {'}</code><br/>
+                                    <code>{'    fmt.Println("Hello, 世界")'}</code><br/>
```

Comment:
For some reason, the indent for this is not displaying in the page in the 'Text entered' column. Might need to use non-breaking spaces